crux-core/src/crux/topology.clj

```diff
@@ -47,9 +47,14 @@
   (let [topology (-> topology
                      (cond-> (not (vector? topology)) vector)
                      (->> (map #(s/conform ::module %))
-                          (apply merge)))]
-    (->> (merge topology (select-keys options (keys topology)))
-         (s/conform ::resolved-topology))))
+                          (apply merge))
+                     (as-> topology (merge topology (select-keys options (keys topology)))))
+        resolved-topology (s/conform ::resolved-topology topology)]
+    (when (s/invalid? resolved-topology)
+      (s/explain ::resolved-topology topology)
+      (throw (IllegalArgumentException. "invalid topology")))
+
```

Comment:
blank line is incongruous? V minor

Reply:
happy to remove this for now, but generally speaking I'd like to see more of these in the codebase - helps the reader break the code down into logical units, and much easier to scan the code when your eye can lock onto blank lines